docs/4-roles-expectations.md

```diff
@@ -1,12 +1,22 @@
 # 4. Roles & Expectations
 
-To operate the guild, five unique roles will be required, each with their own obligations.
+## 4.1 The Protocol Guild
 
-## 4.1 Members
+The Protocol Guild's core output is a curated membership registry, with weights assigned to each member.
 
-There will be three types of members: Slot holders, curators and signers.
+The registry is comprised of individuals who are [actively contributing](https://protocol-guild.readthedocs.io/en/latest/4-roles-expectations.html#qualifications) to Ethereum's core protocol development, while the [weights are calculated](https://protocol-guild.readthedocs.io/en/latest/6-guidelines-for-regular-operation.html#weighting) based on how long each member has been actively contributing.
 
-### 4.11 Slot Holders
+This registry is tied to a [single Ethereum address](https://app.0xsplits.xyz/accounts/0xF29Ff96aaEa6C9A1fBa851f74737f3c069d4f1a9/), which anyone can send funds to, and whose funds will be vested to the membership over time, proportionally to members' normalized weighting.
+
+By providing this registry and address, Ethereum's ecosystem and its community get access to a highly effective and frictionless way to support core protocol contributors. At sufficient scale, this has the downstream effect of boosting the long-term incentives associated with core protocol work.
+
+It should be noted that, apart from providing this registry and facilitating fundraising initiatives related to the registry, the Protocol Guild is not itself responsible for or otherwise involved in Ethereum's core protocol work.
```

Comment:
> related to the registry, Protocol Guild is not itself responsible for or otherwise involved in Ethereum's core protocol work

consider something like:

> It should be noted that Protocol Guild as an entity does not directly manage or participate in the day-to-day stewardship of the Ethereum protocol. This is the responsibility of the existing organizations, teams and individuals whose ongoing focus it is to do so. Similarly, the Guild abstractly as well as its online spaces (eg. twitter, discord, github, etc) are not intended to forums for protocol decision-making, nor is it a bloc to coerce alignment among core contributors. The Guild should be as credibly neutral about this area of protocol stewardship as the Ethereum network is about the usecases built on top of it. Decision making in Ethereum is by rough consensus and should remain so.